Firestore/src/ValueMapper.php

```diff
@@ -402,6 +402,15 @@ private function encodeObjectValue($value)
             return $this->encodeAssociativeArrayValue((array) $value);
         }
 
+        if ($value instanceof \DateTimeInterface) {
+            return [
+                'timestampValue' => [
+                    'seconds' => $value->format('U'),
+                    'nanos' => intval($value->format('u') * 1000)
```

Comment:
Could we use a cast here (and below in the test) instead of a function call? In my testing it can be up to 30% faster.

Reply:
Sorry, I do not understand English very well.

Like that?

`if ($value instanceof \DateTimeInterface) {
	$value = new Timestamp($value);
}`

Reply:
Sorry, I understand now